### PR TITLE
View cart summary front end

### PR DESCRIPTION
**Pull request by helen-choi**
view-cart-summary

### CODE REVIEW

**Comment by helen-choi** [comment]
view-cart-summary

**Comment by helen-choi** [comment]
place-an-order

File: server/index.js [hunk 1]
```diff
@@ -166,6 +166,32 @@ app.post('/api/cart', (req, res, next) => {
...
+    returning *
+    `;
+    const values = [cartId, name, creditCard, shippingAddress];
+
+    db.query(sql, values)
+      .then(result => {
+        res.status(201).json(result.rows[0]);
+        if (result.rows[0] !== undefined) {
+          delete req.session.cartId;
+        }
```
Comment (thebearingedge) [inline server/index.js:190]: If the insert promise is not rejected, then `result.rows` will not be empty. You can simplify this part a bit.

File: server/index.js [hunk 2]
```diff
@@ -166,6 +166,32 @@ app.post('/api/cart', (req, res, next) => {
   }
 });
 
+app.post('/api/orders', (req, res, next) => {
+  const cartId = req.session.cartId;
+  const name = req.body.name;
+  const creditCard = req.body.creditCard;
+  const shippingAddress = req.body.shippingAddress;
```
Comment (thebearingedge) [inline server/index.js:173]: Be sure to verify that these values are present before attempting to query the database.

**Pull request merged**